From 2c79f7218c9aae8a27b0cd7b2fa09a01f6c91db5 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Wed, 1 Apr 2015 00:07:38 +0200
Subject: [PATCH 4/4] Export OpenCV libraries in visp-config, visp.pc and
 VISPConfig.cmake files when OpenCV is installed from ros-<version>-opencv2
 2.4.9 version

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cf5725..b398dde 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1370,7 +1370,9 @@ if(USE_OPENCV)
     #message("OpenCV_LIBS: ${OpenCV_LIBS}")
     #if(WIN32 AND OpenCV_LIB_COMPONENTS AND OpenCV_VERSION AND OpenCV_VERSION VERSION_GREATER 2.4.6.1)
     if(OpenCV_LIB_COMPONENTS AND OpenCV_VERSION AND OpenCV_VERSION VERSION_GREATER 2.4.6.1)
-      set(config_ "NONE" "RELEASE" "DEBUG" "RELEASEWITHDEBINFO") # RelWithDebugInfo was requested under Fedora 20
+      # ReleaseWithDebugInfo was requested to detect libopencv-devel under Fedora 20
+      # RelWithDebugInfo was requested to detect ros-hydro-opencv2 2.4.9 under Ubuntu 12.04 LTS with ROS hydro   
+      set(config_ "NONE" "RELEASE" "DEBUG" "RELEASEWITHDEBINFO" "RELWITHDEBINFO") 
 
       if(POLICY CMP0045)
         # Fix Error on non-existent target in get_target_property for 3rd party location extraction
-- 
1.9.1

